src/data/navigation/sections/tutorials.js

```diff
@@ -0,0 +1,148 @@
+module.exports = [
+    {
+      title: "Introduction",
+      path: "/tutorials/",
+    },
+    {
+      title: "Admin development",
+      path: "/tutorials/admin/",
+      pages: [
+        {
+          title: "Create an Admin page",
```

Comment:
Recommend that the order be

1. Create an Admin page
2. Customize the Admin design
3. Customize the product creation form
4. Create a dynamic row
5. Create a text field attribute